components/blocks/Loader/Loader.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts"></script>
+
+<template>
+  <div class="loader__container">
+    <i-svg-spinner class="loader__spinner" />
+  </div>
+</template>
+
+<style lang="postcss" scoped>
+.loader__container {
+  @apply flex justify-center items-center h-screen w-full;
+}
+
+.loader__spinner {
+  @apply h-20 w-20 fill-current;
+  animation: spin 1.25s steps(9, end) infinite;
+
+  @media (prefers-reduced-motion) {
+    animation-duration: 9s;
+  }
```

Comment:
Just curious about this: is there a reason you set the duration to last eight times to the original? Like is it a best practice or something?

(Feel free to resolve this comment)

Reply:
From [the MDN docs about `prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion):
> The `prefers-reduced-motion` [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media#media_features) is used to detect if a user has enabled a setting on their device to minimize the amount of non-essential motion. The setting is used to convey to the browser on the device that the user prefers an interface that removes, reduces, or replaces motion-based animations.

So, since by default our new `Loader` spinner spins at what I would consider a "normal" rate, for the reduced motion preference I just set it to be slow enough where hopefully it wouldn't be an issue for motion sensitive people while still conveying that the site is doing something.